### PR TITLE
microsoft teams integration documentation

### PR DESCRIPTION
**Pull request by kristianamitchellcontrastsecurity**
This integration is waiting for manager approval first.

### CODE REVIEW

**Comment by kristianamitchellcontrastsecurity** [comment]
@gmoore

**Comment by evamill** [comment]
@kristianamitchellcontrastsecurity How does this relate to the other MS Teams pull request? We have that one on hold right now, so we need to know where this work is at in relation to the Dec 23 release.

**Comment by kristianamitchellcontrastsecurity** [comment]
> @kristianamitchellcontrastsecurity How does this relate to the other MS Teams pull request? We have that one on hold right now, so we need to know where this work is at in relation to the Dec 23 release.

@evamill This is for the official integration with Microsoft Teams the other is not. When the code side of things are established I will tag you in the PR and provide whatever other information you need.

File: content/admin/org_settings/integrations/MicrosoftTeams.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+description: "Integrating MicrosoftTeams with Contrast"
+tags: "Admin organization settings integrations microsoftTeams"
+-->
+
+Use Contrast's Microsoft Teams integration to receive notifications from Contrast in your configured Microsoft Teams instance.
+
+## Setup
+
+### Microsoft Teams
+
```
Comment (gdvdsn-contrast) [inline content/admin/org_settings/integrations/MicrosoftTeams.md]: It looks like our Integrations pages differ in the way they present info about external resources (some provide links, some provide prereqs, some write out the steps). In the future, Docs can develop guidance/templates for this. I think for now, this is OK.

File: content/admin/org_settings/integrations/MicrosoftTeams.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+To add the Contrast integration, go to your team in your Microsoft Teams account, and complete the following steps.
+
+* Choose the appropriate channel to which to send messages, click on the three dots to the right of the channel name, and select **Connectors** from the menu. 
+* In the **Incoming Webhook** row, select **Configure**.
+* Select a name for the Incoming Webhook in the diaglog, and select **Create**.
```
Comment (gdvdsn-contrast) [inline content/admin/org_settings/integrations/MicrosoftTeams.md]: Maybe: Select a name for the Incoming Webhook, and select **Create**.

**Comment by gdvdsn-contrast** [comment]
Hi there - Docs is considering this draft done and ready. Is that the case?

**Comment by gmoore** [comment]
@gdvdsn-contrast If you're done working with it, approve the PR. We'll merge when we ship the feature next week.

**Comment by kristianamitchellcontrastsecurity** [comment]
> Hi there - Docs is considering this draft done and ready. Is that the case?

This is now complete and waiting on approval to merge? @gdvdsn-contrast

**Comment by gdvdsn-contrast** [comment]
This PR has been published here: https://docs.contrastsecurity.com/admin-orgintegrations.html#microsoftteams.

**Pull request merged**